rasa/utils/log_utils.py

```diff
@@ -0,0 +1,84 @@
+import os
+import logging
+import sys
+from typing import Any, Dict
+
+import structlog
+from structlog_sentry import SentryProcessor
+from rasa.plugin import plugin_manager
+
+
+FORCE_JSON_LOGGING = os.environ.get("FORCE_JSON_LOGGING")
+
+
+def _anonymizer(
+    _: structlog.BoundLogger, __: str, event_dict: Dict[str, Any]
+) -> Dict[str, Any]:
+    """Anonymizes event dict."""
+    anonymization_pipeline = plugin_manager().hook.get_anonymization_pipeline()
+
+    if anonymization_pipeline:
+        event_dict["event"] = anonymization_pipeline.log_run(event_dict["event"])
+    return event_dict
+
+
+def configure_structlog() -> None:
+    """Configure logging of the server."""
+    logging.basicConfig(
+        format="%(message)s",
+        stream=sys.stdout,
+        level=logging.DEBUG,
```

Comment:
Let's keep the default as INFO or capture how the user set the logging level from the current CLI options.